website/src/middleware/loggingMiddleware.ts

```diff
@@ -0,0 +1,13 @@
+import { defineMiddleware } from 'astro/middleware';
+
+import { getInstanceLogger } from '../logger.ts';
+
+const logger = getInstanceLogger('RequestLoggingMiddleware');
+
+export const loggingMiddleware = defineMiddleware(async (context, next) => {
+    const response = await next();
+    logger.info(
```

Comment:
Do we really want to log every page load? I think it would be nice on debug.

Reply:
As discussed: For now we'll leave it on `info`, until we notice that it's too much.